index.html

```diff
@@ -1,20 +1,22 @@
 <!doctype html>
   <html>
-    
-   <!--bootstrap-->
-    <link rel="stylesheet" link="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css">
+  <head>
+      <!--bootstrap-->
+       <link rel="stylesheet" link="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css">
+
+       <link rel="stylesheet" href="main.css">
+  </head>
   
     <body>
-<div class="container">
-  <div class="jumbotron"> 
-    <div class="row">
-      <div class="col-xs-12">
-    <h3 class="text-center"><em>The life of</em></h3>
-       <p class="text-center" style="font-size: 34px"><strong>Karl Friedrich Benz</strong></p>
-  <h3 class="text-center"><em>Inventer of the Automobile</em></h3>
+        <div class="container">
+            <div class="jumbotron"> 
+                <div class="row">
+                  <div class="col-xs-12">
+                    <h3 class="text-center"><em>The life of</em></h3>
+                    <p class="text-center" style="font-size: 34px"><strong>Karl Friedrich Benz</strong></p>
+                    <h3 class="text-center"><em>Inventer of the Automobile</em></h3>
    
-   <center><img class="image-shape" src="http://i347.photobucket.com/albums/p456/imjoshuabailey/Karl%20Benz%20with%20Wife_zpslzhcesko.jpg?t=1516294422" alt="Karl Benz takes a ride in an automobile with his wife Bertha">
-    </center>
+                    <center><img class="image-shape" src="http://i347.photobucket.com/albums/p456/imjoshuabailey/Karl%20Benz%20with%20Wife_zpslzhcesko.jpg?t=1516294422" alt="Karl Benz takes a ride in an automobile with his wife Bertha"></center>
```

Comment:
Hmm, I'm not sure center is an HTML tag. Did you mean `<div class="center">CONTENT</div>`?